### ISSUE TITLE
[Feature Request] Total Quantity across all returned records

### ISSUE THREAD

**Issue by jlcanfield** [cleer73/inventory-search#6]
![unknown](https://user-images.githubusercontent.com/40056603/189867172-1e750d04-6ce4-494c-928d-e887659448ee.png)

Maybe instead of "x items found" it could be something like "x unique items found, y total" where x is count of records and y is count of quantity?

**Comment by cleer73** [comment]
Agreed, reading that meta data and seeing those results gives me a few ideas for better search results meta data.

**Comment by cleer73** [comment]
```
Found {unique_character_count} characters, with {total_item_count} total items.
```

Not sure the record count is really relevant, but let me know if you want to know that there are X table records listed in the results.

**Comment by jlcanfield** [comment]
Couldn't hurt.
